File: 06_multilevel_inheritance.py
class Class1:
    def show1(self):
        print("I am class 1")

class Class2(Class1):
    def show2(self):
        print("I am class 2")

class Class3(Class2):
    def show3(self):
        print("I am class 3")


child_obj  =  Class3()
child_obj.show1()
child_obj.show2()
child_obj.show3()

File: 06_multilevel_inheritance.cpp
#include<bits/stdc++.h>
using namespace std;

class Class1{
    public:
    void show1(){
        cout<<"I am class 1"<<endl;
    }
};

class Class2: public Class1{
    public:
    void show2(){
        cout<<"I am class 2"<<endl;
    }
};

class Class3: public Class2{
    public:
    void show3(){
        cout<<"I am class 3"<<endl;
    }
};

int main(){
    
    Class3 child_obj;
    child_obj.show1();
    child_obj.show2();
    child_obj.show3();

    return 0;
}